WordPress/src/main/java/org/wordpress/android/ui/notifications/adapters/NoteBlockAdapter.java

```diff
@@ -48,8 +46,6 @@ public View getView(int position, View convertView, ViewGroup parent) {
         // Update the block type for this view
         convertView.setTag(R.id.note_block_tag_id, noteBlock.getBlockType());
 
-        noteBlock.setBackgroundColor(mBackgroundColor);
-
```

Comment:
Since we're removing this, couldn't we remove the `mBackgroundColor` field and its setter from `NoteBlock` as well?

Reply:
Good catch! We can safely remove it :)